healthcheck/api/src/main/java/org/apache/felix/hc/api/execution/HealthCheckMetadata.java

```diff
@@ -130,6 +132,12 @@ public String getAsyncCronExpression() {
     public Long getAsyncIntervalInSec() {
         return asyncIntervalInSec;
     }
+    
+    /** Return the initial delay for interval in sec used for asynchronous execution.
+     * @return the initial delay for the async interval (or <code>null</code> if not set)  */
```

Comment:
Won't this be `0` instead of `null`, if it is not set? Because the default implementation sets it to `0`.

Reply:
Yes, but the object returned here is `Long` which has been purposefully done to conform to the existing coding convention in Felix HC